### PR TITLE
Add msg command

### PR DESCRIPTION
**Pull request by Zekirax**

### CODE REVIEW

File: src/commands/msg.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+  const private_message = args.slice(1).join(' ')
+
+  const options = {
+    message: JSON.stringify({ text: private_message }),
+    position: Position.CHAT,
+    sender: sender.uuid,
+  }
+
+  world.events.emit('private message', { receiver_username: args[0], options }) // args[0] is the username
```
Comment (DeltaEvo) [inline src/commands/msg.js:54]: Maybe create a `username` variable instead of `args[0] is the username`

File: src/commands/msg.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+              name: 'text',
+              parser: 'brigadier:string',
+              properties: 2,
+            },
+          },
+        ],
+        extraNodeData: {
+          name: 'player',
+          parser: 'minecraft:entity',
+          properties: 0x02, // 0x02 for only player entity
```
Comment (DeltaEvo) [inline src/commands/msg.js:33]: Can you create a constant for that ?

File: src/commands/msg.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    extraNodeData: 'msg',
+    children: [
+      {
+        flags: {
+          command_node_type: 2,
+        },
+        children: [
+          {
+            flags: {
+              command_node_type: 2,
```
Comment (DeltaEvo) [inline src/commands/msg.js:19]: Maybe we should create a constant for that

File: src/player/chat.js [hunk 4]
```diff
@@ -87,5 +87,8 @@ export default {
       )
     })
     world.events.on('chat', (options) => client.write('chat', options))
+    world.events.on('private message', ({ receiver_username, options }) => {
```
Comment (DeltaEvo) [inline src/player/chat.js:90]: Try to avoid using spaces in event names, `private_message` is better

**Review by DeltaEvo — CHANGES_REQUESTED** [review]

File: src/commands/msg.js (hunk 2 above)
Comment (Zekirax) [reply src/commands/msg.js:33]: what is "that"? Be more precise

File: src/commands/msg.js (hunk 3 above)
Comment (Zekirax) [reply src/commands/msg.js:19]: what is "that"? Be more precise

File: src/commands/msg.js (hunk 2 above)
Comment (DeltaEvo) [reply src/commands/msg.js:33]: that, is the line on which I made the comment, but to be more precise, create a constant for the magic number `0x2`

File: src/commands/msg.js (hunk 3 above)
Comment (DeltaEvo) [reply src/commands/msg.js:19]: The `2` constant for the command_node_type

File: src/commands/declare_options.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    ALL: 0x01, // If set, only allows a single entity/player
+    PLAYER: 0x02, // If set, only allows players
+  },
+}
+
+export const command_node_types = {
+  ROOT_INDEX: 0,
+  COMMAND: 1,
+  ARGUMENT: 2,
+}
```
Comment (DeltaEvo) [inline src/commands/declare_options.js:17]: Can you name that with CamelCase so `parser_properties` => `ParserProperties` and `command_node_types` => `CommandNodeTypes`

**Review by DeltaEvo — CHANGES_REQUESTED** [review]
Nice :+1:, just two small little changes :)

File: src/commands/msg.js [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+  },
+]
+
+export default function msg({ world, sender, args }) {
+  if (args.length < 2) {
+    write_error({ sender })
+    return
+  }
+
+  const [username, ...message_words] = args
```
Comment (DeltaEvo) [inline src/commands/msg.js:47]: Perfect :+1:

Comment (Zekirax) [reply src/commands/msg.js:47]: I know. I'm the best.

**Review by DeltaEvo — APPROVED** [review]

**Pull request merged**